packages/customize-widgets/src/components/sidebar-block-editor/use-sidebar-block-editor.js

```diff
@@ -1,14 +1,24 @@
 /**
  * External dependencies
  */
-import { invert, omit, keyBy, isEqual } from 'lodash';
+import { omit, keyBy, isEqual } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { serialize, parse, createBlock } from '@wordpress/blocks';
 import { useState, useEffect, useCallback, useRef } from '@wordpress/element';
 
+function __experimentalAddWidgetIdToBlock( block, widgetId ) {
```

Comment:
This doesn't need to be prefixed with `__experimental` because it's a private function only used in this file.